week-2/F-logical-operators/exercise2.js

```diff
@@ -5,18 +5,34 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number<0) {return true}
+  else {return false}
+}
+
+//function isBetween5and10(number)
```

Comment:
You've commented out the function names here? `//` is a line comment marker.

Also, remember what we said on Saturday about what to do if you find yourself writing:

```
if (something) {
    return true;
} else {
    return false;
}
```
:-) ?